### ISSUE TITLE
Accurate Prediction of Biological Assays with High-Throughput Microscopy Images and Convolutional Networks

### ISSUE THREAD

**Issue by agitter** [gitter-lab/pharmaco-image#15]
https://doi.org/10.1021/acs.jcim.8b00670

> Predicting the outcome of biological assays based on high-throughput imaging data is a highly promising task in drug discovery since it can tremendously increase hit rates and suggest novel chemical scaffolds. However, end-to-end learning with convolutional neural networks (CNNs) has not been assessed for the task biological assay prediction despite the success of these networks at visual recognition. We compared several CNNs trained directly on high-throughput imaging data to a) CNNs trained on cell-centric crops and to b) the current state-of-the-art: fully connected networks trained on precalculated morphological cell features. The comparison was performed on the Cell Painting data set, the largest publicly available data set of microscopic images of cells with approximately 30,000 compound treatments. We found that CNNs perform significantly better at predicting the outcome of assays than fully connected networks operating on precomputed morphological features of cells. Surprisingly, the best performing method could predict 32% of the 209 biological assays at high predictive performance (AUC > 0.9) indicating that the cell morphology changes contain a large amount of information about compound activities. Our results suggest that many biological assays could be replaced by high-throughput imaging together with convolutional neural networks and that the costly cell segmentation and feature extraction step can be replaced by convolutional neural networks.

This looks very relevant to our project.  I believe it is the updated version of the paper from #7.  We should see what their main conclusions are to help us decide what to do next.

I don't necessarily agree with their line in the abstract that looking at how many assays have AUC > 0.9 is the right measure of success.  However, if we review our existing models, do we know how well they do by this metric?
